shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -403,6 +403,8 @@ - (void)viewWillAppear:(BOOL)animated {
     [_engine.get() setViewController:self];
     _engineNeedsLaunch = NO;
   }
+    
+  [self onUserSettingsChanged:nil];
```

Comment:
I'm curious about this one, applicationDidEnterBackground, and applicationWillEnterForeground.

The linked doc doesn't list these - are things not working correctly without them?  It may be worth a short comment about what case these help with even though they're not documented.

Reply:
I think those are remnants of me trying to solve the app switcher problem before I realized what the root cause was. I'll remove them.